### ISSUE TITLE
Negative value of "running_var in batchnorm layer"

### ISSUE THREAD

**Issue by MinLiAmoy** [marvis/pytorch-caffe-darknet-convert#48]
@marvis  I try to transfer the tinyyolo from darknet to pytorch. And when I checked the values of running_var in convolution layer with batchnorm, there were some negative values that only occurred as the first element of a channel. 
I thought **running_var** was supposed to be positive. Did I miss something?

Thanks,
Min

**Comment by MinLiAmoy** [comment]
@marvis
